packages/@aws-cdk-testing/framework-integ/test/aws-dynamodb/test/integ.table-with-customized-role.js.snapshot/iam-policy-report.txt

```diff
@@ -0,0 +1,101 @@
+<AWSServiceRoleForApplicationAutoScaling_DynamoDBTable> (cdk-table-with-customized-role/Table/ScalingRole)
+
+AssumeRole Policy:
+NONE
+
+Managed Policy ARNs:
+NONE
+
+Managed Policies Statements:
+NONE
+
+Identity Policy Statements:
+NONE<my-precreated-role-name> (cdk-table-with-customized-role/Role)
+
+AssumeRole Policy:
+[
+  {
+    "Action": "sts:AssumeRole",
+    "Effect": "Allow",
+    "Principal": {
+      "Service": "dynamodb.amazonaws.com"
+    }
+  }
+]
+
+Managed Policy ARNs:
+NONE
+
+Managed Policies Statements:
+NONE
+
+Identity Policy Statements:
+[
+  {
+    "Action": [
+      "dynamodb:BatchGetItem",
+      "dynamodb:GetRecords",
+      "dynamodb:GetShardIterator",
+      "dynamodb:Query",
+      "dynamodb:GetItem",
+      "dynamodb:Scan",
+      "dynamodb:ConditionCheckItem",
+      "dynamodb:DescribeTable"
+    ],
+    "Effect": "Allow",
+    "Resource": [
+      "(cdk-table-with-customized-role/Table/Resource.Arn)",
+      "(cdk-table-with-customized-role/Table/Resource.Arn)/index/*"
+    ]
+  }
+]<AWSServiceRoleForApplicationAutoScaling_DynamoDBTable> (cdk-table-with-customized-role2/Table/ScalingRole)
```

Comment:
I'm not entirely familiar with it, but is this expected behaviour for the policy report? It seems like here and on L13 and L63 there ought to be newlines separating the different roles.

Reply:
Nice find! These are supposed two be two separate IAM roles and should have newlines between them. I think the original author made a mistake and did not split the lines correctly for multiple roles case. I've fixed it.